framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -2370,6 +2370,197 @@ bool VulkanReplayConsumerBase::CheckCommandBufferInfoForFrameBoundary(
     }
     return false;
 }
+// for Linking Graphics Pipelines Libraries start[--
+#if defined(VK_USE_PLATFORM_ANDROID_KHR)
+std::vector<VkPipeline> libraries(3);
```

Comment:
These global variables and the others below probably need to be keyed at least per-device